.github/workflows/unit-e2e-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: CI Pipeline - Tests Only
+
+on:
```

Comment:
wystarczy tak jak mamy w pierwszym workflow czyli `on: [push]` i potem przed runs-on: jest fajny ifek: `if: github.base_ref != 'develop'` i to załatwia sprawę